### PR TITLE
Generator

### PR DESCRIPTION
**Pull request by Creeperface01**

### CODE REVIEW

File: src/main/java/cn/nukkit/entity/Entity.java [hunk 1]
```diff
@@ -1090,7 +1094,7 @@ public boolean entityBaseTick(int tickDiff) {
             EntityPortalEnterEvent ev = new EntityPortalEnterEvent(this, PortalType.NETHER);
             getServer().getPluginManager().callEvent(ev);
 
-            //TODO: teleport
+            this.setDimension(this.level.getDimension() == Dimension.OVERWORLD ? Dimension.NETHER : Dimension.OVERWORLD);
```
Comment (NycuRO) [inline src/main/java/cn/nukkit/entity/Entity.java]: @Creeperface01 This need add and THE END

Comment (Creeperface01) [inline src/main/java/cn/nukkit/entity/Entity.java]: why?

Comment (Creeperface01) [inline src/main/java/cn/nukkit/entity/Entity.java]: of course, but not here

Comment (NycuRO) [inline src/main/java/cn/nukkit/entity/Entity.java]: @Creeperface01 Why not there?

**Pull request merged**